lib/routes/signin-codes.js

```diff
@@ -51,6 +51,13 @@ module.exports = (log, db, customs) => {
           return db.consumeSigninCode(code)
             .then(result => {
               return request.emitMetricsEvent('signinCode.consumed')
+                .then(() => {
+                  if (result.flowId) {
+                    return request.emitMetricsEvent('flow.continued', {
```

Comment:
I think there's a couple places we emit several right after each other, so if playing, I'd recommend in a separate PR (so we these metrics are delayed either :D).